resources/views/site/blocks/layered_image_viewer.blade.php

```diff
@@ -1,30 +1,43 @@
 @php
-    $items = [];
+    $captionTitle = $block->present()->input('caption_title');
+    $caption = $block->present()->input('caption');
 
-    foreach ($block->childs as $item) {
+    $images = [];
+    $annotations = [];
+
+    foreach ($block->childs as $child) {
         $mediaItem = [
             'type' => 'image',
             'size' => $block->input('size'),
-            'media' => $item->imageAsArray('image', 'desktop'),
-            'label' => $item->input('label'),
+            'media' => $child->imageAsArray('image', 'desktop'),
+            'label' => $child->input('label'),
         ];
 
-        $items[] = $mediaItem;
+        if ($child['type'] == 'layered_image_viewer_img') {
+            $images[] = $mediaItem;
+        }
+
+        if ($child['type'] == 'layered_image_viewer_annotation') {
+            $annotations[] = $mediaItem;
+        }
     }
 @endphp
 
-@if (count($items) > 0)
+@if (count($images) > 0 || count($annotations) > 0)
     @component('components.organisms._o-layered-image-viewer')
         @slot('variation', 'o-blocks__block')
-        @slot('items', $items)
+        @slot('captionTitle', $captionTitle)
+        @slot('caption', $caption)
+        @slot('images', $images)
+        @slot('annotations', $annotations)
         @slot('imageSettings', $imageSettings ?? array(
-                'srcset' => array(200,400,600,1000,1500,3000),
+                'srcset' => array(300,600,800,1200,1600,2000,3000,4500),
```

Comment:
Could you provide some rationale behind changing these numbers?

Reply:
The srcset was previously modelled off of the gallery block, but then I decided to model it off of the media molecule instead. I think this could do with some more testing though to make sure we’re using the most appropriate srcset, so I’ve created a new Pivotal story to make sure this is picked up.